docs/logistics/getting_help.md

```diff
@@ -8,6 +8,17 @@ You can ask anything on the Slack `#ohw21-general` channel, any time. Organizers
 
 We've set up the `#ohw21-helpdesk` Slack channel to ask technical questions (or other types of questions, if you'd like). We will set up some helpdesk Slack user group that you can use to tag your question as, say, about Python. We'll list and describe those groups here. You are also welcome to direct message the helpers if you don't feel comfortable asking questions publicly.
```

Comment:
Please edit this text to change it from "we will set up", "we'll list", etc, to a present tense